src/dumpulator/details.py

```diff
@@ -528,3 +542,23 @@ def __setitem__(self, index, value):
     "Reserved"
 ]
 assert len(interrupt_names) == 32
+
+flagposdict = {

```

Comment:
I think it would be better to move this into the Registers class directly to not dirty the global namespace.